index.js

```diff
@@ -224,4 +224,43 @@ window.onload = () => {
   contents.forEach((content) => {
     observer.observe(content);
   });
+
+  //   document.querySelector("input").onchange = () => {
+  //     alert("hi")
+  //   }
```

Comment:
- [ ] Please kindly remove this commented lines of code to make your work clean. 👍